src/object/pushbutton.cpp

```diff
@@ -62,15 +63,17 @@ HitResponse
 PushButton::collision(GameObject& other, const CollisionHit& hit)
 {
   auto player = dynamic_cast<Player*>(&other);
-  if (!player) return FORCE_MOVE;
+  auto rock = dynamic_cast<Rock*>(&other);
+  if (!player && !rock) return FORCE_MOVE;
```

Comment:
Please add a line break. (`return` jumps out of the function execution, so it should always stick out.)
```suggestion
  if (!player && !rock) 
    return FORCE_MOVE;
```